fabric-transitive-access-wideners-v1/build.gradle

```
@@ -223,6 +225,21 @@ def generateCreatorEntityTrackedDataMethods(List<String> lines, FileSystem fs) {
 	generateTrackedDataFields("net/minecraft/entity/decoration/InteractionEntity", lines, fs, "shouldRespond")
 }
 
+def generateEnchantmentMethods(List<String> lines, FileSystem fs) {
+	lines.add("# Private methods of Enchantment and EnchantmentHelper")
+
+	for (def node : [ loadClass(fs.getPath("net/minecraft/enchantment/EnchantmentHelper.class")), loadClass(fs.getPath("net/minecraft/enchantment/Enchantment.class")) ]) {
+		for (def method : node.methods) {
+			// All private inner methods of EnchantmentHelper
+			if ((method.access & Opcodes.ACC_PRIVATE) != 0 && !method.name.startsWith("method_")) {
+				lines.add("transitive-accessible method $node.name $method.name $method.desc")
+			}
+		}
+	}
+	lines.add("transitive-accessible class net/minecraft/enchantment/EnchantmentHelper\$Consumer")
+	lines.add("transitive-accessible class net/minecraft/enchantment/EnchantmentHelper\$ContextAwareConsumer")
```

Nit: no need to escape the `$` if you use normal single-quoted strings instead of interpolating double-quoted strings:
```suggestion
	lines.add('transitive-accessible class net/minecraft/enchantment/EnchantmentHelper$Consumer')
	lines.add('transitive-accessible class net/minecraft/enchantment/EnchantmentHelper$ContextAwareConsumer')
```